rclpy/src/rclpy/_rclpy_action.c

```diff
@@ -16,6 +16,7 @@
 
 #include <rcl/error_handling.h>
 #include <rcl_action/rcl_action.h>
+#include <rcutils/strdup.h>
```

Comment:
This is probably not needed now?

Reply:
Yep, totally correct.  Removed in 3038019